third_party/lua-cjson/lua_cjson.c

```diff
@@ -432,6 +434,12 @@ static void json_append_data(lua_State *l, struct luaL_serializer *cfg,
             const char *str = field.errorval->errmsg;
             return json_append_string(cfg, json, str, strlen(str));
         }
+        case MP_DATETIME:
+        {
+            char buf[DT_TO_STRING_BUFSIZE];
+	    size_t sz = tnt_datetime_to_string(field.dateval, buf, sizeof(buf));
+	    return json_append_string(cfg, json, buf, sz);
```

Comment:
This block is misaligned. Last 2 lines somewhy have tabs in the beginning, but in this file the indentation is spaces.

Reply:
Fixed

Reply:
I suppose somebody had hope that we might want to integrate new changes from the upstream. But IMO it is diverged too much already. We could reformat this file entirely.